### DISCUSSION TITLE
CMD - README

### DISCUSSION THREAD

**Discussion by FidelTachini** [dude333/rapina#32] [Q&A]
Boa noite dude333,

Acabei de ver seu post no Reddit sobre essa ferramenta que desenvolveu. Tenho um background de zero experiência com programação e tentei executar os passos descritos na seção readme, mas sem sucesso.

Ao abrir o CMD existe outro comando pertinente a ativação do arquivo que baixei no release? Porque executo ./rapina get e não tenho  nada além da resposta : "'.' não é reconhecido como um comando interno ou externo, um programa operável ou um arquivo em lotes."

Desde já agradeço.

**Comment by dude333** [comment]
Oi @FidelTachini, 

Vamos lá ver se consigo te ajudar :)

1. Baixar a última versão do rapina na área de Releases: https://github.com/dude333/rapina/releases. Para o Windows versão 64-bits, temos o [rapina-v0.13.1-windows-4.0-amd64.exe](https://github.com/dude333/rapina/releases/download/v0.13.1/rapina-v0.13.1-windows-4.0-amd64.exe).

2. Abrir o **cmd**. Você provavelmente  vai cair na pasta `C:\Users\voce` (onde *voce* é o seu usuário). Localizar onde o arquivo acima (`rapina-v0.13.1-windows-4.0-amd64.exe`) foi salvo e ir até lá. O arquivo provavelmente foi salvo na pasta `Downloads`, então, se for o caso, é só entrar ali com o comando `cd Downloads`.

3. Rodar o executável `rapina-v0.13.1-windows-4.0-amd64.exe` e pronto. Agora sim pode seguir os passos do README.

Opcional:
1. Mover o `rapina-v0.13.1-windows-4.0-amd64.exe` para outro diretório.
2. Renomear este executável para `rapina.exe` para ficar mais fácil a digitação :)

Espero que tenha ajudado.

**Reply by FidelTachini** [reply]
Ajudou e muito! 

Fico profundamente agradecido pelo seu auxílio e pelo programa que desenvolveu. Vai me ajudar imensamente para definição de novos investimentos.

Seu trabalho ficou muito legal.

Deixo novamente meus agradecimentos e parabéns pelo que fez.
